views/components/user-status/macro.njk

```diff
@@ -1,17 +1,8 @@
 {% macro userStatus(user) %}
     {% if user %}
-        <div class="user-status">
```

Comment:
Where has this part gone? Is this in the new user nav now?

Reply:
Yeah, that nav defaults to this same info (logged-in username and logout link).
![image](https://user-images.githubusercontent.com/394376/60083170-1e594480-972d-11e9-98cb-23b6c7e1a466.png)